ansible/roles/dspace/files/solr.in.sh

```diff
@@ -0,0 +1,271 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this file seems to be a duplicated of _ansible/roles/solr/files/solr.in.sh_
-> please use just one file (task is also duplicated)